pkg/controller/storageprofile-controller.go

```diff
@@ -140,7 +139,7 @@ func (r *StorageProfileReconciler) getStorageProfile(sc *storagev1.StorageClass)
 
 func (r *StorageProfileReconciler) reconcilePropertySets(sc *storagev1.StorageClass) []cdiv1.ClaimPropertySet {
 	claimPropertySets := []cdiv1.ClaimPropertySet{}
-	capabilities, found := storagecapabilities.Get(sc)
+	capabilities, found := Get(r.client, sc)
```

Comment:
should we maybe rename the func to something more specific like GetStorageCapabilities? kind of weird to just say Get() here